mysql-test/suite/tokudb/r/recovery.result

```diff
@@ -0,0 +1,31 @@
+create table t (a int, b int, primary key (a)) engine=tokudb;
+insert into t values (1,2),(2,4),(3,8);
+optimize table t;
+Table	Op	Msg_type	Msg_text
+test.t	optimize	note	Table does not support optimize, doing recreate + analyze instead
+test.t	optimize	status	OK
+# restart
+insert into t values (4,9),(10,11),(12,13);
+select * from t;
+a	b
+1	2
+2	4
+3	8
+4	9
+10	11
+12	13
+# Kill the server
+# restart:--tokudb-force-recovery=6 --read-only --super-read-only
+select * from t;
+a	b
+1	2
+2	4
+3	8
+# restart:--tokudb-force-recovery=6 --read-only --super-read-only
```

Comment:
Can we test the behavior if --tokudb-force-recovery=6 and --read-only --super-read-only is not specified?  What happens?  If --read-only --super-read-only are not specified and --tokudb-force-recovery=6 is specified, I think we should fail to initialize the plugin.

Reply:
Okay, I'll add that. (We open the tables in read only mode anyway with force recovery, so my guess would be that we get errors elsewhere in the code)